doc/hotspot-style.html

```diff
@@ -1116,7 +1116,7 @@ <h3 id="noexcept">noexcept</h3>
 href="http://wg21.link/n3050">n3050</a>) are permitted with restrictions
 described below.</p>
 <ul>
-<li>Only the abbreviated form of <code>noexcept</code> exception
+<li>Only the argument-less form of <code>noexcept</code> exception
 specifications are permitted. <code>noexcept</code> exception
 specifications with arguments are forbidden.</li>
```

Comment:
I was suggesting dropping the second sentence as it is implied by the first.

Reply:
Oh, I see.  Sure.  Fixed.